Dockerfile

```diff
@@ -1,3 +1,17 @@
+# Copyright 2020-2021 Dave Verwer, Sven A. Schmidt, and other contributors.
```

Comment:
> and other contributors.

![](https://media.giphy.com/media/wCG00aS1PTQys/giphy.gif)

Reply:
I'm glad someone noticed this!